java/lang/padding_oracle_encryption_vulnerability.yml

```diff
@@ -0,0 +1,40 @@
+patterns:
+  - pattern: $<CIPHER>.getInstance($<PADDING_ORACLE>)
+    filters:
+      - variable: CIPHER
+        regex: ^(javax.)?(crypto.)?Cipher
+      - variable: PADDING_ORACLE
+        string_regex: .*\/CBC\/PKCS5Padding
```

Comment:
```suggestion
        string_regex: \/CBC\/PKCS5Padding\z
```

Reply:
Ah, this isn't matching as expected - the same for `\A)RSA|ECIES)`. Maybe we could look together on this one?